### DISCUSSION TITLE
Hexo d报错Error: src must be a string or an object describing the source

### DISCUSSION THREAD

**Discussion by xiaohe123awa** [hexojs/hexo#5386] [Question / Answer]
![C445F98E-0A46-4CD4-B2AC-77DD6137E250](https://github.com/hexojs/hexo/assets/115407956/d1e6a739-341e-4caf-9b42-474544df52bd)
如图，求助
Error: src must be a string or an object describing the source

**Comment by passwordgloo** [comment]
I met the same problem just now , but I solve it . This error means your input content is invalid . Here is my solution :
- Check your input content recently , sometimes  some  braces  such as `<` 、`[` will affect deploy
- Check your media files address especially local address ;
- Atthough hitespace characters can not see , they can also influence deployments

I wish my answer can help you get out of the woods .
